RefactoringAndItsPurpose/Practice/RefactoringAndItsPurposePractice/test/IdentifyIssuesInCodeSnippetTest.kt

```diff
@@ -55,8 +52,4 @@ class IdentifyIssuesInCodeSnippetTest : BaseIjTestClass() {
             "Please, rename \"w\" parameter to \"width\""
         }
     }
-
-    private fun hasParameter(parameterName: String) = ApplicationManager.getApplication().runReadAction<Boolean> {
```

Comment:
Don't we need this check anymore?

Reply:
This has been put into the test framework